tools/cldr-apps/src/main/java/org/unicode/cldr/web/UserRegistry.java

```diff
@@ -521,8 +526,10 @@ JSONObject getPermissionsJson() throws JSONException {
                     .put("userCanUseVettingParticipation", userCanUseVettingParticipation(this))
                     .put("userIsAdmin", userIsAdmin(this))
                     .put("userIsManager", getLevel().canManageSomeUsers())
-                    .put("userIsTC", userIsTC(this))
-                    .put("userIsVetter", userIsVetter(this) && !userIsTC(this))
+                    .put("userIsTC", userIsTCOrStronger(this))
+                    // Caution: userIsVetter really means user is Vetter or Manager, but not TC or
+                    // Admin. It should be renamed to avoid confusion.
+                    .put("userIsVetter", userIsVetterOrStronger(this) && !userIsTCOrStronger(this))
```

Comment:
tech debt; I only added a comment for now

Reply:
didn't you already rename it? so the comment is out of date?

Reply:
No, I renamed the method to `userIsVetterOrStronger`, but the json item is still `userIsVetter`

The json item needs a name like `userIsVetterOrStrongerButNotTCOrStronger` or `userIsExactlyVetterOrManager` -- such a change is beyond the scope of this ticket though